src/panels/config/helpers/forms/show-dialog-schedule-block-info.ts

```diff
@@ -3,6 +3,7 @@ import { fireEvent } from "../../../../common/dom/fire_event";
 export interface ScheduleBlockInfo {
   from: string;
   to: string;
+  data: Record<string, any>;
```

Comment:
Are you sure about this type? I think it can be undefined. Also in the core PR the example is with a number `data: 71`